waltz-ng/client/data-flow/data-flow-edit.js

```diff
@@ -84,7 +84,7 @@ function controller($q,
                     application,
                     authSourceStore,
                     dataFlowStore,
-                    dataTypeStore,
+                    dataTypesService,
```

Comment:
That looks like a typo, i'll sort that

On Thu, Aug 11, 2016 at 5:25 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-ng/client/data-flow/data-flow-edit.js
> <https://github.com/khartec/waltz/pull/417#discussion_r74455312>:
>
> > @@ -84,7 +84,7 @@ function controller($q,
> >                      application,
> >                      authSourceStore,
> >                      dataFlowStore,
> > -                    dataTypeStore,
> > +                    dataTypesService,
>
> maybe we should move back to singular naming: dataTypeService
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74455312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC452yGLj21MVgTap6Jn1TRiC5-ZhiHvks5qe0zqgaJpZM4JiBOI>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk